### PR TITLE
Moved 'Forgot password' below input area on sign-in prompt

### LINKED ISSUE #4109: Move "forgot password" link below/after password input
### Description

When logging in with your GitHub account, the next tab stop after username is "Have you forgotten your password" or similar wording. It should probably be the password input instead.

~~Also, if you have 2FA enabled it would be nice if the code input was focused.~~ Moved to #4132

### Version

**GitHub Desktop version:** 1.0.13
**OS version:** Microsoft Windows [Version 10.0.16299.248]

### Steps to Reproduce

1. Open GitHub Desktop
2. File -> Options
3. Sign in (only tested github.com, not enterprise)

### PR DESCRIPTION
**Pull request by iamnapo**
Fixes #4109

Before:

After:

cc @nerdneha (I can't mention @desktop/design 😛)

### CODE REVIEW

File: app/src/ui/sign-in/sign-in.tsx [hunk 1]
```diff
@@ -203,10 +203,16 @@ export class SignIn extends React.Component<ISignInProps, ISignInState> {
...
-            labelLinkText="Forgot password?"
-            labelLinkUri={state.forgotPasswordUrl}
           />
         </Row>
+        <Row>
+          <LinkButton
+            className="forgot-password-link-sign-in"
+            uri={state.forgotPasswordUrl}
+          >
+            Forgot password
```
Comment (niik) [inline app/src/ui/sign-in/sign-in.tsx]: Seems like the question mark got dropped when moving this, could we bring it back please?

Comment (iamnapo) [reply app/src/ui/sign-in/sign-in.tsx]: Actually, I did it on purpose to be on par with [@app/src/ui/lib/authentication-form.tsx#L124](https://github.com/desktop/desktop/blob/master/app/src/ui/lib/authentication-form.tsx#L124). Should I add one there, too?

Comment (niik) [reply app/src/ui/sign-in/sign-in.tsx]: Yes, I'd really appreciate that!

**Comment by iamnapo** [comment]
😎

**Comment by tierninho** [comment]
tested = ✅

**Pull request merged**